misc-scripts/xref_mapping/XrefParser/MGIParser.pm

```diff
@@ -111,12 +86,11 @@ sub run {
 	}
       }
       $count++;
-    }
-    else{
-      print STDERR "PROBLEM: $line";
-    }
 
   }
+  $input_file->eof or croak "Error parsing file $file: " . $input_file->error_diag();
+  $file_io->close();
+
```

Comment:
This might be a matter of preference but the parsers I have seen so far would only print the summary in verbose mode.